ops/docker/Dockerfile

```diff
@@ -1,10 +1,12 @@
 #syntax=docker/dockerfile:1
-FROM node:16-bullseye-slim as builder
-RUN apt-get update -y && apt-get install -y git
+FROM node:18-bullseye-slim as builder
```

Comment:
we might use node 20 or higher like 22

Reply:
No we can't, there were some package(s) (I forgot which one, but some of them are setting engine version to <= 18) having issues on a higher node version, our code base is too old.